output/text.go

```diff
@@ -96,6 +96,16 @@ func Text(results []*comparer.Result, includePassing bool, tweaks []*config.Quer
 	fmt.Printf("Total: %d / %d (%.2f%%) passed, %d unsupported\n", successes, len(results), 100*float64(successes)/float64(len(results)), unsupported)
 
 	fmt.Printf("Duration Ref prom: %v, Kfuse: %v\n", impl1Duration, impl2Duration)
+
+	if targetTotalDuration != "" {
+		duration, err := time.ParseDuration(targetTotalDuration)
+		if err != nil {
+			fmt.Printf("Error: Unable to parse input target total duration")
+		}
+		if impl2Duration > duration {
```

Comment:
Let's print both ref/kfuse duration.

Where is the test failed if we exceed this duration?

Reply:
Ref and Kfuse duration is already printed in line 98.

I'm thinking we can do the actual failure in circleci, similar to how we check the number of passes in circleci.